in_toto/models/layout.py

```diff
@@ -203,6 +193,36 @@ def import_step_metadata_from_files_as_dict(self):
 
     return step_link_dict
 
+
+@attr.s(repr=False, init=False)
+class _Layout(models__common.Signable):
```

Comment:
What about a more descriptive name, like e.g. `LayoutSignable`, or `LayoutPayload`? What do you think?

Reply:
That's probably clearer in the long term. I adopted this approach because of the `_` prefix, but it's probably not that clear/useful. Will change.